### PR TITLE
Move setlanguage to controller

### PR DESCRIPTION
**Pull request by rullzer**
Ajax: :fire: 

CC: @LukasReschke @nickvergessen @MorrisJobke

### CODE REVIEW

File: settings/Controller/PersonalController.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	private $l;
+
+	/**
+	 * PersonalController constructor.
+	 *
+	 * @param string $appName
+	 * @param IRequest $request
+	 * @param IFactory $l10nFactory
+	 * @param $userId
+	 * @param IConfig $config
```
Comment (LukasReschke) [inline settings/Controller/PersonalController.php]: IL10N is missing

**Review by LukasReschke — APPROVED** [review]
Looks good to me. Little PHPDoc nitpick though :)

File: settings/Controller/PersonalController.php (hunk 1 above)
Comment (rullzer) [inline settings/Controller/PersonalController.php]: Always something to complain ;)
Fixed.

**Comment by MorrisJobke** [comment]
Tested and works 👍

**Pull request merged**